### PR TITLE
Visual tweaks from vojta

### PR DESCRIPTION
**Pull request by AkashGutha**

### CODE REVIEW

File: src/Components/PricingTierCard.js [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+                background: "#222",
+                marginTop: "-34px",
+                transform: "skewY(18deg)",
+                transformOrigin: "top left",
+                marginLeft: "260px"
+              }
+            })}
+          >
+            <p className={css({ fontSize: "30px" })}>SAVE 50%</p>
+            <p className={css({ fontSize: "15px" })}>usually 299,99$</p>
```
Comment (tayiorbeii) [inline src/Components/PricingTierCard.js]: `.` instead of `,` for the price?

File: src/Pages/LandingPage.js [hunk 2]
```diff
@@ -13,14 +14,30 @@ import TopicsTabs from "./../Components/TopicsTabs";
 const LandingPageObserver = observer(
   class LandingPage extends Component {
     navigateToArticle = id => {
-      console.log("object");
       this.props.history.push("/article/" + id);
     };
 
     componentWillReact() {
       console.log("I will re-render, since the observables changed!");
```
Comment (tayiorbeii) [inline src/Pages/LandingPage.js]: remove log

File: src/Pages/LandingPage.js [hunk 3]
```diff
@@ -13,14 +14,30 @@ import TopicsTabs from "./../Components/TopicsTabs";
...
+    componentDidMount() {
+      window.addEventListener("resize", this.updateDimensions);
+    }
+    componentWillUnmount() {
+      window.removeEventListener("resize", this.updateDimensions);
+    }
+
+    updateDimensions = () => {
+      this.setState({ width: window.innerWidth });
+      console.log("dimensions updated");
```
Comment (tayiorbeii) [inline src/Pages/LandingPage.js]: remove log

File: src/Pages/LandingPage.js [hunk 4]
```diff
@@ -13,14 +14,30 @@ import TopicsTabs from "./../Components/TopicsTabs";
...
+      console.log(this.state);
```
Comment (tayiorbeii) [inline src/Pages/LandingPage.js]: remove log

**Review by tayiorbeii — CHANGES_REQUESTED** [review]
There are some `no-unused-vars` warnings when running locally. 

The vertical edges of the logo icon are cut off a bit: https://cl.ly/1S040w183I22

The search button on the article detail page isn't centered quite right: https://cl.ly/2N1w2x1I1h0I

Also, I'm thinking that the article URLs with the big long UUIDs are temporary, but just pointing them out :)

File: src/Components/PricingTierCard.js (hunk 1 above)
Comment (AkashGutha) [reply src/Components/PricingTierCard.js]: new `pricing page` is up so nothing to bother about this.

**Comment by tayiorbeii** [comment]
👍

**Pull request merged**